s4/clarity/configuration/udf.py

```diff
@@ -102,3 +102,16 @@ def _add_preset_internal(self, preset_value):
         preset_node = ETree.SubElement(self.xml_root, 'preset')
         preset_node.text = types.obj_to_clarity_string(preset_value)
 
+
+class Field(WrappedXml):
```

Comment:
This class is a bit confusing because it isn't really a field definition, the Udf class above is. It's more of a reference to a Udf. In an ideal world this would be dealt with as we deal with other references, in that it resolves to a Udf object. Since these references are by name that would require some interesting changes.